xmtp_proto/src/api_client.rs

```diff
@@ -166,3 +173,23 @@ pub trait XmtpMlsClient: Send + Sync + 'static {
         request: SubscribeWelcomeMessagesRequest,
     ) -> Result<WelcomeMessageStream, Error>;
 }
+
+// Wasm futures don't have `Send` or `Sync` bounds.
+#[cfg_attr(target_arch = "wasm32", async_trait(?Send))]
+#[cfg_attr(not(target_arch = "wasm32"), async_trait)]
+pub trait XmtpIdentityClient: Send + Sync + 'static {
+    async fn publish_identity_update(
+        &self,
+        request: PublishIdentityUpdateRequest,
+    ) -> Result<PublishIdentityUpdateResponse, Error>;
+
+    async fn get_identity_updates_v2(
```

Comment:
Needed to make this a `v2` since we already have a `get_identity_updates` method on the client for the old endpoint.

Reply:
To clarify, `get_identity_updates_v2` is getting xmtp v3 identities right?